docs/content/2.deploy/providers/firebase.md

```diff
@@ -78,15 +96,102 @@ You can find more details in the [Firebase documentation](https://firebase.googl
 You can preview a local version of your site if you need to test things out without deploying.
 
 ```bash
-NITRO_PRESET=firebase yarn build
+NITRO_PRESET=firebase npm run build
```

Comment:
In some places I just used npm to run commands to not add to much noise about switching the package manager each time as people should know what they use anyway